app/controllers/dashboard_controller.rb

```diff
@@ -13,7 +13,8 @@ def main
   def staff
     @departments = Department.includes :positions
     @pending_interviews = Interview.pending.group_by(&:position)
-    @pending_records = ApplicationRecord.pending.newest_first
+    @pending_records = ApplicationRecord.where(saved_for_later: false)
```

Comment:
Should we also provide a way to see the applications that have been saved for later?  I see something to that effect in the `PositionsController`, but perhaps a dashboard option or link might work too?